### PR TITLE
Feature/killmails

### PR DESCRIPTION
**Pull request by herpaderpaldent**
wow this has become so easy with the way this package is build right now.

### CODE REVIEW

File: src/Notifications/KillMail/AbstractKillMailNotification.php [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+        return true;
+    }
+
+    /**
+     * @return bool
+     */
+    final public static function isPersonal(): bool
+    {
+
+        return false;
```
Comment (warlof) [inline src/Notifications/KillMail/AbstractKillMailNotification.php]: This overload is unneeded as it's default value 😜

File: src/Notifications/KillMail/AbstractKillMailNotification.php [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+        $cached_entry = cache('name_id:' . $id);
+
+        if (! is_null($cached_entry))
+            return $cached_entry;
+
+        if ($is_alliance)
+            return $this->getAllianceTicker($id);
+
+        // Resolve the Esi client library from the IoC
+        $eseye = app('esi-client')->get();
```
Comment (warlof) [inline src/Notifications/KillMail/AbstractKillMailNotification.php]: Need a better way to do this and avoid to spam esi for every single sent notification.

Comment (herpaderpaldent) [reply src/Notifications/KillMail/AbstractKillMailNotification.php]: Ups forgot the cache ....

File: src/Notifications/KillMail/AbstractKillMailNotification.php (hunk 1 above)
Comment (herpaderpaldent) [reply src/Notifications/KillMail/AbstractKillMailNotification.php]: I can't confirm this. Doesn't contain the default. https://github.com/herpaderpaldent/seat-notifications/blob/24a1c7b99eb07fa4cccaab6bec863692342c7328/src/Notifications/KillMail/AbstractKillMailNotification.php also i believe it is the right place to define which notifications are present

**Pull request merged**